web_external/Featuresets/FeaturesetsView.js

```diff
@@ -24,6 +24,11 @@ isic.views.FeaturesetsView = isic.View.extend({
         }, this);
         this.featuresets.fetch();
 
+        // TODO: Use the more general 'update' event, once Backbone is upgraded from Girder
+        this.listenTo(this.featuresets, 'remove', function () {
+            this.render();
+        }, this);
```

Comment:
Yeah, I actually did that in a place inside the `FeaturesetView`, but didn't see it here.

Also, note the line above this one is also incorrectly passing `this`. That enticed me to do the same here. I'll fix both.